### DISCUSSION TITLE
[Sovol Klipper Screen] Find help, and help others.

### DISCUSSION THREAD

**Discussion by bassamanator** [bassamanator/Sovol-SV06-firmware#173] [Announcements]
Please post fixes, mods, etc. here. Thanks!

- Some useful [links](https://github.com/bassamanator/Sovol-SV06-firmware/discussions/172#discussioncomment-12255285).

**Comment by JamesHabben** [comment]
I created a fix for the timezone issue on the Sovol Klipper screens. It is here if you would like to see it.

https://github.com/JamesHabben/SovolStuff

**Comment by nubecoder** [comment]
I don't think this has received enough attention yet.

Free the Sovol Klipper Screen from outdated SW:
https://github.com/vasyl83/SV07update
